static/app/components/modals/debugFileCustomRepository/http.tsx

```diff
@@ -58,6 +58,7 @@ type Props = Pick<ModalRenderProps, 'Header' | 'Body' | 'Footer'> & {
 
 function Http({Header, Body, Footer, onSubmit, ...props}: Props) {
   const initialData: Data = {
+    id: props.initialData?.id ?? uniqueId(),
```

Comment:
iiuc this is the actual fix of the issue right?

Reply:
Yes, it is. I can also split these two apart if that makes things easier to roll out.